tests/integration/test_unary_ufunc.py

```diff
@@ -50,10 +50,9 @@ def check_result(op, in_np, out_np, out_num, **isclose_kwargs):
     )
     if not result:
         print(f"cunumeric.{op} failed the test")
-        print("Inputs:")
-        for arr in in_np:
-            print(arr)
-            print(f"dtype: {arr.dtype}")
+        print("Input:")
+        print(in_np)
```

Comment:
So now `in_np` is always just one array or a scalar?

Reply:
Correct, all the test cases were a single input (unary functions) so I just simplified this to be just one array.